### PR TITLE
Update README with detailed information about the toolkit

### PR DESCRIPTION
**Pull request by AmongUsafk**
Update `README.md` to provide a detailed description of the Nano11 Toolkit.

* **Introduction**: Add an overview of the Nano11 Toolkit, highlighting its purpose and functionalities.
* **Features**: List key functionalities such as customization, optimization, management, updates, and user-friendly interface.
* **Installation**: Provide step-by-step instructions for downloading, extracting, and running the application.
* **Usage**: Include instructions on how to launch and use the application, as well as checking for updates.
* **Contributing**: Add guidelines for contributing to the project, including forking the repository, creating branches, making changes, and submitting pull requests.

### CODE REVIEW

**Comment by AmongUsafk** [comment]
TO DO :
- [ ] Add The `Contributation.md` for contributation GuideLines

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+## Introduction
+
+Nano11 Toolkit is a powerful and versatile tool designed to enhance your Windows experience. It provides a range of functionalities to customize, optimize, and manage your Windows operating system with ease. Whether you are a power user or a casual user, Nano11 Toolkit offers features that cater to your needs.
+
+## Features
+
+- **Customization**: Personalize your Windows experience with various customization options.
+- **Optimization**: Improve system performance with optimization tools.
+- **Management**: Manage installed applications and system settings efficiently.
+- **Updates**: Keep your system up-to-date with the latest updates and patches.
```
Comment (StarNumber12046) [inline README.md]: Updates could be removed

**Review by StarNumber12046 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AmongUsafk) [reply README.md]: ok

**Review by StarNumber12046 — APPROVED** [review]
Okay!

**Pull request merged**